### PR TITLE
App secret

### PR DESCRIPTION
**Pull request by mentalcaries**
# Describe your changes 

At this PR I created the secret key for the app.
Updates to environment variables to follow in Slack channel

# I follow best practices 

- [ ] My code is covered by unit tests 
- [ ] This PR is less than [300 lines of code](https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests)
- [ ] My code [smells good](https://en.wikipedia.org/wiki/Code_smell) 
- [ ] I have tested my PR locally and proven it works. 
- [ ] There is [no copy-paste](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ dotenv.config();
 // app.use(logger('dev'));
 
 // Required environment variables for development and production
-const { MONGODB_URI, FRONTEND_URL } = process.env;
+const { DOTCOT_SECRET, MONGODB_URI, FRONTEND_URL } = process.env;
```
Comment (McCambley) [inline backend/app.js:27]: Something like "SESSION_SECRET" might be a little more descriptive, as these are all sort of "DOTCOT_SECRETS", but that's not a big deal at all. This looks good, thanks for hiding the things!

**Review by McCambley — APPROVED** [review]
One comment below, but looks good! Approved!

**Review by McCambley — CHANGES_REQUESTED** [review]
Forgot to add in the previous review, and realized I could resubmit another review. This variable should be added to the `.yml` files to be used in ci/cd stuff (testing and deployment). Vlad should also add this secret to the repositories secrets.

File: backend/app.js (hunk 1 above)
Comment (mentalcaries) [reply backend/app.js:27]: I like it

File: .github/workflows/CI-only.yml [hunk 2]
```diff
@@ -26,6 +26,8 @@ jobs:
       FACEBOOK_TEST_APP_SECRET: ${{secrets.FACEBOOK_TEST_APP_SECRET}}
       LINKEDIN_CLIENT_ID: ${{secrets.LINKEDIN_CLIENT_ID}}
       LINKEDIN_CLIENT_SECRET: ${{secrets.LINKEDIN_CLIENT_SECRET}}
+      DOTCOT_SECRET: ${{secrets.DOCTCOT_SECRET}}
```
Comment (McCambley) [inline .github/workflows/CI-only.yml:29]: 👍

**Review by McCambley — APPROVED** [review]
Looks good to me. Once @vgalilei adds the secret to the settings in this repository, we should start seeing tests pass.

**Comment by vgalilei** [comment]
We have broken tests in the main branch. I think tests will not pass even if I add this to the env. 
I rather advise putting some fake keys directly to the CI configuration than adding to secrets. Because it is not secret while we run tests

**Pull request merged**